### DISCUSSION TITLE
Location of development build and version number of build

### DISCUSSION THREAD

**Discussion by HanOostdijk** [duckdb/duckdb-spatial#194] [Q&A] [answered]
In README.md it states under [development-builds](https://github.com/duckdb/duckdb_spatial#development-builds) that one 
> can also grab the lastest builds directly from the CI runs or the release page here on GitHub and install manually.

Because I am not familiar with the building process, it would be useful for me when 
the location of the builds is also explicitly mentioned in README.md

Another question: is it possible to inspect the 'version number' of the extension in DuckDB with an SQL statement?

**Comment by Maxxen** [comment, marked as answer]
Hi! The CI runs are found in the `Actions` tab here on GitHub, and builds are produced by the `Main Extension Distribution Pipeline` job. For example, if you go [here](https://github.com/duckdb/duckdb_spatial/actions/runs/6954255607) and scroll aaaaalll the way down, there are download links. 

Im working on a big docs update, but you can also install the latest built extension version for the latest stable duckdb version (e.g. v0.9.2) by executing: 

```sql
FORCE INSTALL spatial FROM 'http://nightly-extensions.duckdb.org';
```

There is no way to check the exact version of an extension yet though.

**Reply by HanOostdijk** [reply]
Thanks for the quick response. Han
